src/clj/harja/palvelin/ajastetut_tehtavat/paivystystarkistukset.clj

```diff
@@ -36,7 +36,8 @@
                                (format "Harja: Urakalta %s puuttuu päivystystiedot päivämäärälle %s"
                                        urakka-nimi
                                        (fmt/pvm (c/to-date pvm)))
-                               (viesti-puuttuvasta-paivystyksesta urakka-nimi pvm))))
+                               (viesti-puuttuvasta-paivystyksesta urakka-nimi pvm)
+                               {})))
```

Comment:
laheta-viesti! rajapintaan on lisätty parametri http-headereitä varten. Näin tämän lopulta yksiselitteisenä ja yksinkertaisempana vaihtoehtona, kuin esim alkaa kikkailla multiarityillä, kun käyttöjä oli lopulta varsin rajallinen määrä. Uusi rajapinta on siis alla, ja tuo tyhjä mäppi annetaan jos ei ole headereitä mitä halutaan passata, ja se sietää myös nil:n.

  (laheta-viesti!
   [this lahettaja vastaanottaja otsikko sisalto headers]